templates/web/greenwich/header_logo.html

```diff
@@ -15,15 +15,4 @@
         <li><a href="/">FixMyStreet</a></li>
     </ul>
 
-    <ul id="fms-menu-desktop">
-        <li><[% IF c.req.uri.path == '/' %]span[% ELSE %]a href="/"[% END %] class="report-a-problem-btn"
-            >[% loc("Report a problem") %]</[% c.req.uri.path == '/' ? 'span' : 'a' %]></li>[%
-        %]<li><[% IF c.req.uri.path == '/my' %]span[% ELSE %]a href="/my"[% END
-            %]>[% loc("Your reports") %]</[% c.req.uri.path == '/my' ? 'span' : 'a' %]></li>[%
-        %]<li><[% IF c.req.uri.path == '/reports' %]span[% ELSE %]a href="/reports"[% END
-            %]>[% loc("All reports") %]</[% c.req.uri.path == '/reports' ? 'span' : 'a' %]></li>[%
-        %]<li><[% IF c.req.uri.path == '/alert' %]span[% ELSE %]a href="/alert[% pc ? '/list?pc=' : '' %][% pc | uri %]"[% END
-            %]>[% loc("Local alerts") %]</[% c.req.uri.path == '/alert' ? 'span' : 'a' %]></li>[%
-        %]<li><[% IF c.req.uri.path == '/faq' %]span[% ELSE %]a href="/faq"[% END
-            %]>[% loc("Help") %]</[% c.req.uri.path == '/faq' ? 'span' : 'a' %]></li>
-    </ul>
+    [% INCLUDE "main_nav.html" body_name=c.cobrand.council_name hide_mysoc_link=1 ul_class="fms-menu-desktop" omit_wrapper=1 %]
```

Comment:
Good catch - this commit was from before the `ul_id` (mandated by Warwickshire's CSS) was added.